pyhf/readxml.py

```diff
@@ -26,7 +28,7 @@ def import_root_histogram(rootdir, filename, path, name):
 
     raise KeyError('Both {0:s} and {1:s} were tried and not found in {2:s}'.format(name, os.path.join(path, name), os.path.join(rootdir, filename)))
 
-def process_sample(sample,rootdir,inputfile, histopath, channelname):
+def process_sample(sample,rootdir,inputfile, histopath, channelname, enable_tqdm=False):
     if 'InputFile' in sample.attrib:
```

Comment:
here and in the other cases, I'd also suggest renaming to `track_progress` instead of `enable_tqdm`